### PR TITLE
microphone stuck fixes

### PR DESCRIPTION
**Pull request by chrisnojima**
This doesn't fix this 100%. I think how it works currently is a little flakey but doing it a different way would be a much larger project which i don't think we should do now. I believe its mostly issues around gesture handling that launches things on top of itself and there being timing issues w/ lost taps.

This pr makes it harder to repro for me, mostly by turning off taps of things that don't need taps and additionally making the 'slide to cancel' text actually clickable so you can actually escape the state where its stuck thinking you're holding the tap down

### CODE REVIEW

File: shared/chat/audio/audio-starter.native.tsx [hunk 1]
```diff
@@ -131,6 +132,7 @@ const AudioStarter = (props: AudioStarterProps) => {
         simultaneousHandlers={panRef}
       >
         <Kb.PanGestureHandler
+          shouldCancelWhenOutside={false}
```
Comment (mmaxim) [inline shared/chat/audio/audio-starter.native.tsx:135]: What does this help with?

Comment (chrisnojima) [reply shared/chat/audio/audio-starter.native.tsx:135]: https://software-mansion.github.io/react-native-gesture-handler/docs/handler-common.html#shouldcancelwhenoutside

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by thebearjew** [comment]
@chrisnojima Should we ticket the larger project and back burner it? So it's not forgotten?

**Comment by chrisnojima** [comment]
Unclear. If you're using this normally I think it's hard to repro

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by thebearjew** [comment]
@chrisnojima testing this after standup

**Pull request merged**